homeassistant/components/notify/group.py

```diff
@@ -61,11 +63,17 @@ def __init__(self, hass, entities):
 
         tasks = []
         for entity in self.entities:
+            service_template = entity.get(CONF_SERVICE_TEMPLATE)
```

Comment:
Each template needs to get `hass` set once during initialization, that has not happened.